.github/workflows/migration.yml

```diff
@@ -0,0 +1,46 @@
+name: "Test migrations"
+
+on:  
+  pull_request:
+    branches:
+      - "**"
```

Comment:
Maybe this could be scoped to run only on PRs that changes the content of the migrations folder, on other PR there is no reason to check this.

Reply:
I agree that would be a good further optimization. I started this PR to first implement the testing procedure